src/components/curriculum/CurriculumBootcamp.js

```diff
@@ -43,22 +45,30 @@ const CurriculumBootcamp = ({
     type,
     isOpen,
   }
+
+  const meetupTime = '16:30 - 18:30'
+  const trainingTime = '09:00 - 18:30'
+
   const firstHalf = (
     <React.Fragment>
       <H4>Course outline:</H4>
       <Section
         {...commonProps}
         title="Evening pre-bootcamp (optional)"
         name="day0"
-        subTitle="React 101 and JS fundamentals"
+        subTitle={`React 101 and JS fundamentals (${TrainingDateByDay({
+          training,
+        })}  ${meetupTime})`}
```

Comment:
@ric9176 the one in August should display 6:30 pm to 9 pm. what about:
```
// TODO remove this DEPRECATED_ logic after July 2019
const DEPRECATED_IS_JULY2019 = isJuly2019(startDate);
const meetupTime = DEPRECATED_IS_JULY2019?  '16:30 - 18:30': '6:30 - 9 pm'
```

Reply:
👍